postgres/osmap.yaml

```diff
@@ -91,9 +91,70 @@ RedHat:
 {% endif %}
 
 Suse:
+  pkg_repo:
+    name: pgdg-sles-{{ release }}
+    humanname: PostgreSQL {{ repo.version }} $releasever - $basearch
+    #Suse repos are suffixed 'suse/sles-12-$basearch' by upstream community convention.
+    baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/suse/sles-12-$basearch'
```

Comment:
I cannot use '$releasever' because packages target Suse family (SUSE 12+, OpenSUSE 42+), so the convention today seems to be the hardcoding of "sles-12".  Using '$releasever' breaks the formula for Suse family except Sles 12.

Reply:
Yeah, that's true, I understand. My point was that the installation could be broken on other SUSE variants which is not currently supported by upstream: SLES 11, OpenSuse 13 or Tumbleweed.
In such cases it's better to fail fast. But that also introduces complexities.
I think this is fine for now.